src/log/log.h

```diff
@@ -125,6 +125,50 @@ struct __wt_txn_printlog_args {
     uint32_t flags;
 };
 
+struct __wt_log_manager {
+
+/* AUTOMATIC FLAG VALUE GENERATION START 0 */
+#define WT_LOG_CONFIG_ENABLED 0x001u  /* Logging is configured */
+#define WT_LOG_DOWNGRADED 0x002u      /* Running older version */
+#define WT_LOG_ENABLED 0x004u         /* Logging is enabled */
+#define WT_LOG_EXISTED 0x008u         /* Log files found */
+#define WT_LOG_FORCE_DOWNGRADE 0x010u /* Force downgrade */
+#define WT_LOG_INCR_BACKUP 0x020u     /* Incremental backup log required */
+#define WT_LOG_RECOVER_DIRTY 0x040u   /* Recovering unclean */
+#define WT_LOG_RECOVER_DONE 0x080u    /* Recovery completed */
+#define WT_LOG_RECOVER_ERR 0x100u     /* Error if recovery required */
+#define WT_LOG_RECOVER_FAILED 0x200u  /* Recovery failed */
+#define WT_LOG_REMOVE 0x400u          /* Removal is enabled */
+#define WT_LOG_ZERO_FILL 0x800u       /* Manually zero files */
+                                      /* AUTOMATIC FLAG VALUE GENERATION STOP 32 */
+    uint32_t flags;                   /* Global logging configuration */
```

Comment:
Typically in WT, the flags field is at the end of the structure. I think it helps for readability and consistency with other structures.